src/transaction.cpp

```diff
@@ -1672,6 +1687,33 @@ int wsrep::transaction::certify_commit(
 
     assert(lock.owns_lock());
 
+    lock.unlock();
```

Comment:
Same as in `certify_fragment()` could this be moved below, after changing the state to certifying?

Reply:
done